pkg/cli/image/info/info.go

```diff
@@ -400,7 +400,7 @@ func (o *ImageRetriever) Run() error {
 						return callbackFn(name, nil, imagemanifest.NewImageForbidden(msg, err))
 					}
 					if imagemanifest.IsImageNotFound(err) {
-						msg := fmt.Sprintf("image %q does not exist", from)
+						msg := fmt.Sprintf("image %q not found: %s", from, err.Error())
 						return callbackFn(name, nil, imagemanifest.NewImageNotFound(msg, err))
```

Comment:
Is it worth updating `NewImageNotFound` to suffix non-`nil` errors?  It just [serializes as the `msg` today][1], but updating it would mean we'd get the `err`-level detail in all of our call-sites, not just this one.  Although it's only one other call-site which has a chance at a non-`nil` error:

```console
$ git --no-pager grep '\.NewImageNotFound('
pkg/cli/image/extract/extract.go:                                               return imagemanifest.NewImageNotFound(msg, err)
pkg/cli/image/info/info.go:                                             return callbackFn(name, nil, imagemanifest.NewImageNotFound(msg, err))
pkg/cli/image/info/info.go:                                     return imagemanifest.NewImageNotFound(fmt.Sprintf("no manifests could be found for %q", from), nil)
```

[1]: https://github.com/openshift/oc/blob/9eb2f8f9d5b573ffeb720f17bc45cc8507f861bd/pkg/cli/image/manifest/errors.go#L17-L18

Reply:
yup I did notice there are others, I'll update